desktop/storeService.ts

```diff
@@ -1,11 +1,26 @@
-const Store = require('electron-store');
+import { nativeTheme } from 'electron';
+import Store from 'electron-store';
+import { TypedSchemaStore } from '../shared/types';
 
 class StoreService {
-  static store: typeof Store;
+  public static store: Store<TypedSchemaStore>;
 
   static init() {
     if (!StoreService.store) {
-      StoreService.store = new Store();
+      StoreService.store = new Store<TypedSchemaStore>({
+        accessPropertiesByDotNotation: true,
+        defaults: {
+          accounts: {},
+          userSettings: {
+            darkMode: nativeTheme.shouldUseDarkColors
```

Comment:
please follow convention - all boolean vars should be prefixed with 'is'

Reply:
I'd say that any boolean flag should sound like a question:
E.G. `isDarkMode`, `isValid`, `hasOddNumbers`, `didChanged`

Reply:
Fixed.